[2022-01-06 15:07:06,338] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: new_description.print_date manual__2022-01-06T09:36:59.764087+00:00 [queued]>
[2022-01-06 15:07:06,346] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: new_description.print_date manual__2022-01-06T09:36:59.764087+00:00 [queued]>
[2022-01-06 15:07:06,346] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-06 15:07:06,346] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-06 15:07:06,346] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-06 15:07:06,357] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): print_date> on 2022-01-06 09:36:59.764087+00:00
[2022-01-06 15:07:06,359] {standard_task_runner.py:52} INFO - Started process 58141 to run task
[2022-01-06 15:07:06,366] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'new_description', 'print_date', 'manual__2022-01-06T09:36:59.764087+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/new_description_dag.py', '--cfg-path', '/var/folders/3z/fyvld5jd6lxbb7nt49h4jx1h0000gn/T/tmp13e1c7fy', '--error-file', '/var/folders/3z/fyvld5jd6lxbb7nt49h4jx1h0000gn/T/tmpkqf86mby']
[2022-01-06 15:07:06,368] {standard_task_runner.py:77} INFO - Job 29: Subtask print_date
[2022-01-06 15:07:06,412] {logging_mixin.py:109} INFO - Running <TaskInstance: new_description.print_date manual__2022-01-06T09:36:59.764087+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-01-06 15:07:06,452] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=new_description
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2022-01-06T09:36:59.764087+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-06T09:36:59.764087+00:00
[2022-01-06 15:07:06,453] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/3z/fyvld5jd6lxbb7nt49h4jx1h0000gn/T
[2022-01-06 15:07:06,454] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-01-06 15:07:06,461] {subprocess.py:85} INFO - Output:
[2022-01-06 15:07:06,468] {subprocess.py:89} INFO - Thu Jan  6 15:07:06 IST 2022
[2022-01-06 15:07:06,468] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-06 15:07:06,487] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=new_description, task_id=print_date, execution_date=20220106T093659, start_date=20220106T093706, end_date=20220106T093706
[2022-01-06 15:07:06,512] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-06 15:07:06,536] {local_task_job.py:264} INFO - 2 downstream tasks scheduled from follow-on schedule check
